src/core/Notification/InlineNotification/InlineNotification.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { Component, forwardRef } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlDiv, HtmlDivWithRef } from '../../../reset';
+import { Icon } from '../../Icon/Icon';
+import { Heading } from '../../Heading/Heading';
+
+import {
+  BaseNotificationProps,
+  notificationClassNames,
+  baseClassName,
+} from '../BaseNotification/BaseNotification';
+import { AutoId } from '../../utils/AutoId/AutoId';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../theme';
+import { baseStyles } from './InlineNotification.baseStyles';
+
+export interface InlineNotificationProps extends BaseNotificationProps {
+  /** Label for the Notification */
+  labelText: string;
```

Comment:
In the designs the success notification (which is what I think the InlineNotification is for) has no labelText, so we can't have that as a required prop. The need for it in general should probably be discussed.

Reply:
Fixed labelText to be optional.